res/mqtt-export/configuration.toml

```diff
@@ -103,18 +103,18 @@ Port = 6379
 Timeout = "30s"
```

Comment:
need to remove `ConfigAccessTokenFile ` and `AccessTokenFile ` from the above config.